src/darwin/CHIPTool/CHIPTool/View Controllers/DeviceSelector.m

```diff
@@ -43,7 +43,7 @@ - (void)refreshDeviceList
 {
     uint64_t nextDeviceID = CHIPGetNextAvailableDeviceID();
     _deviceList = [NSMutableArray new];
-    for (uint64_t i = 0; i < nextDeviceID; i++) {
+    for (uint64_t i = 1; i < nextDeviceID; i++) {
```

Comment:
why the change in indexing here? Seems unrelated to persistence logic ... is it?

C/C++ array indexing from 1 is usually odd. Could you add a comment explaining?

Reply:
Yeah I can do that. Node ID 0 is reserved and so there's no sense is looking for a device with that ID. I should have left a comment.